enumeratum-core/src/test/scala/enumeratum/Models.scala

```diff
@@ -227,6 +227,19 @@ object UncapitalisedEnum extends Enum[UncapitalisedEnum] {
 
 }
 
+case object MultiEnum extends Enum[MultiEnum] {
+  import scala.collection.breakOut
+  val values = findValues
```

Comment:
You can't do this because the type parameter passed to `Enum` must be a sealed trait.

Compilation error https://travis-ci.org/lloydmeta/enumeratum/jobs/343456716#L844

```scala
[error] /home/travis/build/lloydmeta/enumeratum/enumeratum-core/src/test/scala/enumeratum/Models.scala:232:16: You can only use findValues on sealed traits or classes
[error]   val values = findValues
[error]                ^
```

Reply:
Fixed.